ansible/.env.j2

```diff
@@ -0,0 +1,15 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+TRELLO_API_KEY={{trello_api_key}}
+TRELLO_API_TOKEN={{trello_api_token}}
+TRELLO_API_OPEN_LIST=65b7e01862e855c31fa82dd2
+TRELLO_API_CLOSED_LIST=65b839408b3682b30a0aedf0
+TRELLO_API_BOARD=6597c2327b717082e2ed3429
```

Comment:
This works nicely for us, but worth considering whether it's worth extracting these into variables like the API key/token - we could even set `default` arguments so that we don't need to type them in each time, but still making it parameterized so that we can easily deploy into a different environment later if we want to